examples/vols/ex.yml

```diff
@@ -0,0 +1,34 @@
+version: 0.1-dev
+
+services:
+- name: db
+  containers:
+  - image: mysql
+    mounts:
+    - volumeName: db
+      mountPath: /var/lib/mysql
+
+- name: backup
+  containers:
+  - image: backup
+    mounts:
+    - volumeName: db
+      mountPath: /app/store
+      volumeSubPath: foo/bar
```

Comment:
I have no idea, unfortunately, quite a newbie on kubernetes but I know from related-ish projects like:

https://github.com/BD2KGenomics/toil
https://github.com/common-workflow-language/common-workflow-language

That users are gonna expect it to be supported :-S